parsons-tool-frontend/src/pages/ProblemEvaluation.js

```diff
@@ -15,6 +15,7 @@ export default function ProblemEvaluation() {
   const { nextProblem, currentProblem } = useProblems();
 
   const [problem, setProblem] = useState(null);
+  const [isFaded, setIsFaded] = useState(false);
```

Comment:
You could also probably just do something like
```
const isFaded = curentProblem.problem.blocks.some(block => block && block.fadedIndices.length > 0);
```

Reply:
sure will try it out